### ISSUE TITLE
Soil ID fails at all lat/longs

### ISSUE THREAD

**Issue by DerekCaelin** [techmatters/terraso-mobile-client#1772]
### App Version

169

### Platforms

- [X] iOS
- [X] Android
- [ ] iOS simulator
- [ ] Android emulator

### Description

When I tap any point on the map, a callout appears with spinners indicating a query to the LandPKS web service. All points return an error on my android and iOS devices. Another device running 169 can successfully return SoilID. 

### Steps to Reproduce

1. On the sites screen, tap any location. For example: 
`39.53032, -105.04067`

### Expected behavior

After a wait, the site will return a soil ID. 

### Actual behavior

The spinner will go for 20s, then return an Error

### Additional context

On another device (Steve's) running 169, `39.53032, -105.04067` returns a Soil ID result.

**Comment by knipec** [comment]
Hmm I wasn't able to repro this in the morning, but now on my iOS device I seem to be running into it. 
Here are two Sentry issues that it appears are from the event (based on my IP and the timing): 
- [TypeError: Network request failed](https://techmatters.sentry.io/issues/5504281568/?project=4506299761623040&query=is%3Aunresolved+issue.priority%3A%5Bhigh%2C+medium%5D+user.ip%3A172.92.175.216&referrer=issue-stream&sort=date&statsPeriod=24h&stream_index=0)
- [SyntaxError: JSON Parse error: Unexpected character: <](https://techmatters.sentry.io/issues/5504862811/?project=4506299761623040&query=is%3Aunresolved+issue.priority%3A%5Bhigh%2C+medium%5D+user.ip%3A172.92.175.216&referrer=issue-stream&sort=date&statsPeriod=24h&stream_index=1)

Also, I closed the app and re-opened it again, clicked on a location, and successfully got soil ID matches. Elevation showed a spinner indefinitely (several minutes).  
After a few minutes, I clicked on another location and got errors. Elevation took a while to get, but eventually populated. 
Shortly after, I clicked on another location and successfully got soil ID matches. Elevation showed a spinner indefinitely. 


On Android I have had some locations with errors for soil ID.
One location (47.57198, -122.30058) gave the Error box on the location's Soil ID screen, and unexpectedly with no prompting changed to the real soil ID response.
Sometimes elevation showed a spinner indefinitely, but sometimes it showed a value within a second.
